sumologic/resource_sumologic_s3_data_forwarding_destination.go

```diff
@@ -0,0 +1,146 @@
+package sumologic
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceSumologicS3DataForwardingDestination() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSumologicS3DataForwardingDestinationCreate,
+		Read:   resourceSumologicS3DataForwardingDestinationRead,
+		Update: resourceSumologicS3DataForwardingDestinationUpdate,
+		Delete: resourceSumologicS3DataForwardingDestinationDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
```

Comment:
Same comment here to rename this to destination_name to have similar naming convention with api